web/src/main/java/com/futurewei/alcor/web/entity/route/NeutronRouterWebRequestObject.java

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.web.entity.route;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.common.entity.CustomerResource;
+import lombok.Data;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+import java.util.List;
+
+@Data
+public class NeutronRouterWebRequestObject extends CustomerResource {
+
+    @JsonProperty("routetable")
+    private RouteTable routeTable;
+
+    // store vpc_id
+    @JsonProperty("owner")
+    private String owner;
+
+    // store subnet_gateway_port_id / interface
+    @JsonProperty("ports")
+    private List<String> ports;
+
+    @JsonProperty("tenant_id")
+    private String tenantId;
+
+    @JsonProperty("admin_state_up")
```

Comment:
@xieus Do we need admin_state_up attribute for VPC router?
'admin_state_up' is for neutron router actually. If we don't need it in VPC, we can put this attribute in the neutron specific attribute object which is router_extra_attributes object.

Reply:
This depends on whether users could leverage this field in the VPC scenario, and how we plan to use this field. 

The answer to the first question could be NO as user may not have visibility to a VPC router. 

For the second question, can admin use this field to check L3 routing status or something else?

Reply:
For VPC, I think the router is implicit for user, even for the admin, since one VPC can only has one router. So, admin_state or router_state for VPC is meaningless. In VPC's L3 routing, most of operations are depends on routing tables, but in Neutron's case, they are depends on router.

Reply:
Combined the fields in one class could make our model cover both vpc and neutron router,  if there is no other effect